common/src/main/java/xyz/wagyourtail/jsmacros/client/api/library/impl/FWorld.java

```diff
@@ -323,7 +216,7 @@ public String getBiome() {
         assert mc.world != null;
         return mc.world.getRegistryManager().get(Registry.BIOME_KEY).getId(mc.world.getBiome(mc.player.getBlockPos())).toString();
     }
-    
+
```

Comment:
fix your CodeStyle so this doesn't happen... it's a bunch of bloat on the diff.

Reply:
he removed your whitespace

Reply:
I tried to restore as many as I could.

Reply:
And you may want to remove them yourself, as well as optimize the imports, because it's generally not good practice to use wildcard and unused imports, as well as having intendents on empty lines.